getting_started/step_by_step/your_first_game.rst

```diff
@@ -140,35 +143,59 @@ node, so we'll add a script. Click the ``Player`` node and click the
 In the script settings window, you can leave the default settings alone. Just
 click "Create":
 
+.. note:: If you're creating a C# script or other languages, select the 
+            language from the `language` drop down menu before hitting create.
+
 .. image:: img/attach_node_window.png
 
 .. note:: If this is your first time encountering GDScript, please read
           :ref:`doc_scripting` before continuing.
 
 Start by declaring the member variables this object will need:
 
-::
+.. tabs::
+ .. code-tab:: gdscript GDScript
 
     extends Area2D
 
     export (int) var SPEED  # how fast the player will move (pixels/sec)
     var screensize  # size of the game window
+ 
+ .. code-tab:: csharp
+
+    public class Player : Area2D
+    {
+        [Export] public int SPEED = 0;
```

Comment:
To follow the coding convention they should be `Speed` and `screenSize` respective, but it seems their GDScript counterparts were inconsistent in naming at the first place.

Maybe we should change GDScript part as well?